packages/dds/tree/src/shared-tree/schematizingTreeView.ts

```diff
@@ -184,6 +188,7 @@ export class SchematizingSimpleTreeView<
 					policy: this.schemaPolicy,
 				},
 				this,
+				true, // if isInitialization is true, schema validation does not occur
```

Comment:
Why don't we validate in the initialize case?

When documenting code, it's more important to put in a comment for why it's doing something instead of just what it's doing. A comment about what is hard to know if it's out of date. A comment about why provides new useful information, which can be used to understand if the code is doing the right thing.